nuxeo-runtime/nuxeo-launcher-commons/src/test/java/org/nuxeo/launcher/config/ConfigurationGeneratorTest.java

```diff
@@ -88,6 +88,13 @@ public void getJavaOptsWithConfig() throws Exception {
     }
 
     @Test
+    public void getJavaOptsWithMultivaluedProperty() throws Exception {
```

Comment:
`getJavaOptsWithMultivaluedProperty` ; wasn't the issue related to the use of quotes in a Java option?

Reply:
This is not related to the resolution of the ticket. I just want to ensure on Java side, `ConfigurationGenerator` can handle multi-valued property correctly. I can remove it if it is confusing.